test_conformance/common/vulkan_wrapper/opencl_vulkan_wrapper.cpp

```diff
@@ -1191,14 +1191,15 @@ cl_external_memory_handle_type_khr vkToOpenCLExternalMemoryHandleType(
 {
     switch (vkExternalMemoryHandleType)
     {
+        default:
```

Comment:
We may want to do something (like log an error message?) in the default case, since this is almost certainly a bug that should be fixed.

Reply:
Added log.